src/components/note/note.spec.tsx

```diff
@@ -256,63 +256,87 @@ describe("Note", () => {
     });
 
     it("throws if the width is < 0", () => {
       expect(() => {
-        render({ width: 0 });
+        renderNote({ width: 0 });
       }).toThrow("<Note> width must be greater than 0");
     });
 
     it("throws if createdDate is not defined", () => {
       expect(() => {
-        render({ createdDate: undefined });
+        renderNote({ createdDate: undefined });
       }).toThrow("<Note> createdDate is required");
     });
 
     it("throws if noteContent is not defined", () => {
       expect(() => {
-        render({ noteContent: undefined });
+        renderNote({ noteContent: undefined });
       }).toThrow("<Note> noteContent is required");
     });
 
     it("throws if status.text is not defined", () => {
       expect(() => {
-        render({ status: {} });
+        renderNote({ status: {} });
       }).toThrow("<Note> status.text is required");
     });
 
     it("throws if status.timeStamp is not defined", () => {
       expect(() => {
-        render({ status: { text: "Edited" } });
+        renderNote({ status: { text: "Edited" } });
       }).toThrow("<Note> status.timeStamp is required");
     });
 
     it("throws if inlineControl is not an ActionPopover", () => {
       expect(() => {
-        render({ inlineControl: <button type="button">A Button</button> });
+        renderNote({ inlineControl: <button type="button">A Button</button> });
       }).toThrow("<Note> inlineControl must be an instance of <ActionPopover>");
     });
   });
 
   describe("Link Previews", () => {
-    it("renders any LinkPreviews passed in via the `previews` prop", () => {
+    it("renders any LinkPreviews passed in via the `previews` prop passed as an array", () => {
       const previews = [
-        <LinkPreview url="foo" />,
-        <LinkPreview url="bar" />,
-        <LinkPreview url="wiz" />,
+        <LinkPreview url="foo" key="foo" />,
+        null,
+        <LinkPreview url="bar" key="bar" />,
+        undefined,
+        <LinkPreview url="wiz" key="wiz" />,
       ];
-      const wrapper = render({ previews });
+      const wrapper = renderNote({ previews });
+      expect(wrapper.find(LinkPreview).length).toEqual(3);
+      wrapper
+        .find(LinkPreview)
+        .forEach((preview) => expect(preview.find("a").exists()).toBeTruthy());
+    });
+
+    it("renders any LinkPreviews passed in via the `previews` prop passed as a node", () => {
+      const previews = (
+        <>
+          <LinkPreview url="foo" key="foo" />,
+          <LinkPreview url="bar" key="bar" />,
+          <LinkPreview url="wiz" key="wiz" />,
+        </>
+      );
+      const wrapper = renderNote({ previews });
       expect(wrapper.find(LinkPreview).length).toEqual(3);
       wrapper
         .find(LinkPreview)
         .forEach((preview) => expect(preview.find("a").exists()).toBeTruthy());
     });
   });
 });
+
+describe("coverage filler for required but unused draft-js Editor internal onChange prop", () => {
```

Comment:
It might be better to use `istanbul ignore` on the related line of code instead of this test as we it will be harder to refactor this when we convert to RTL